### DISCUSSION TITLE
OBC windows for Position instead of Drivers

### DISCUSSION THREAD

**Discussion by Shiro-Z31** [f1multiviewer/issue-tracker#501] [Feature requests]
During a race I manually keep 8 OBCs for top 8 positions on screen. It would be cool if windows could be assigned to a position and the stream automatically change when passing happens.

I'd also like to see OBC camera window highlight somehow when that driver radio is active. sometimes it's hard to track who's talking.

![PXL_20240309_170424031](https://github.com/f1multiviewer/issue-tracker/assets/162834769/145c92dc-3501-4fb1-aacd-93df1edc4748)

**Comment by jeppeolesen** [comment]
Duplicate of #63
